lib/commands/find.js

```diff
@@ -35,7 +35,17 @@ helpers.findElOrEls = async function (strategy, selector, mult, context = '') {
     try {
       element = await this.doFindElementOrEls(params);
     } catch (err) {
-      if (err instanceof errors.NoSuchElementError) {
+
+      // if the error that comes back is from a proxied request, we need to
+      // unwrap it to its actual protocol error first
+      if (_.isFunction(err.getActualError)) {
+        err = err.getActualError(); // eslint-disable-line no-ex-assign
+      }
+
+      // now we have to inspect the error to determine if it is a no such
+      // element error, based on the shape of the error object from
+      // appium-base-driver
+      if (err.jsonwpCode === 7 || err.error === "no such element") {
```

Comment:
why should we use magic numbers here? I'd rather use `errors.NoSuchElementError.error()/code()`

Reply:
right you are! will fix